security/tpm/samples/SecurityProviderTpmSimulator/SecurityProviderTpmSimulator.csproj

```diff
@@ -2,10 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <RootNamespace>Microsoft.Azure.Devices.Provisioning.Security</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU1605,NU1902,NU1903</NoWarn>
```

Comment:
these are warnings as errors and are regarding older version of newtonsoft being used.
latest version is 13 and we use 12